mozilla/donate-wagtail
# updated fields to have white/grey background

Closes #1139 
Related PRs/issues #


**Steps to test**:

1. Visit https://donate-wagta-firefox-io-8lpxaj.herokuapp.com/en-US/
2. Click on the credit card button to land on the details form
3. If everything looks as expected, testing is complete! 

Note: I have visited the review app on my IOS device using firefox and nightmode, and can confirm that the fix is working as intended 😄
